### DISCUSSION TITLE
Supported Data Sources

### DISCUSSION THREAD

**Discussion by narennadig** [NovatecConsulting/novatec-service-dependency-graph-panel#63] [Q&A] [answered]
Hey,
Is there any option to add elastic as a data source in the future??
Also what are the data sources currently supported?

**Comment by MariusBrill** [comment, marked as answer]
Hi @narennadig, 
you can use any datasources provided by grafana. You can read [here](https://github.com/NovatecConsulting/novatec-service-dependency-graph-panel#use-your-own-data-source) how the data needs to be formatted.

Greetings
Marius
